cvat/apps/dataset_manager/annotation.py

```diff
@@ -746,6 +746,10 @@ def interpolate(shape0, shape1):
             curr_frame = shape["frame"]
             prev_shape = shape
 
+            # keep at least 1 shape
+            if end_frame <= curr_frame:
```

Comment:
If we put the condition before iteration code (a bit changed):

```python
if shape["frame"] >= end_frame:
    break
```

We probably do not need additional check in `` cvat/apps/dataset_manager/bindings.py``. Right?

Reply:
I'm not sure if interpolation is the only source of this problem, so I'd better keep the check in `bindings.py` anyway. I wouldn't remove this check in the interpolation function above, because the code using this function definitely expects at least 1 frame in a track, so putting it before loop would break this convention.